oas

```diff
@@ -856,8 +842,7 @@
                   "description": "Date-time that the application of water should performed. The date MUST be the same as the recommendation date"
```

Comment:
I don't understand this change. Before, we could have one recommendation per date (for as may days as you want), and for each date, the times had to be different and the ratios had to total 100. Now, there can be only one overall recommendation which could be performed in applications over multiple days? Is that really what you want?